server/portal/apps/workspace/api/views.py

```diff
@@ -247,21 +247,31 @@ def post(self, request, *args, **kwargs):
 
             if settings.DEBUG:
                 wh_base_url = settings.WH_BASE_URL + '/webhooks/'
-                # jobs_wh_url = settings.WH_BASE_URL + reverse('webhooks:jobs_wh_handler')
+                jobs_wh_url = settings.WH_BASE_URL + reverse('webhooks:jobs_wh_handler')
             else:
                 wh_base_url = request.build_absolute_uri('/webhooks/')
-                # jobs_wh_url = request.build_absolute_uri(reverse('webhooks:jobs_wh_handler'))
+                jobs_wh_url = request.build_absolute_uri(reverse('webhooks:jobs_wh_handler'))
 
             job_post['parameterSet']['envVariables'] = job_post['parameterSet'].get('envVariables', []) + [{'key': '_webhook_base_url', 'value':  wh_base_url}]
 
             portal_name = settings.PORTAL_NAMESPACE
             job_post['tags'] = job_post.get('tags', []) + [portal_name]
 
-            # TODOv3 Webhooks/notifications continues
-            # job_post['notifications'] = [
-            #     {'url': jobs_wh_url,
-            #      'event': e}
-            #     for e in settings.PORTAL_JOB_NOTIFICATION_STATES]
+            # ttlMinutes of 0 corresponds to max default (1 month)
```

Comment:
My understanding was ttl of `0` was unlimited, but leaving blank would take the tapis default of 1 month. Is that right?

Reply:
Not quite. Steve mentioned that even though the Notification service accepts 0 as unlimited, the job service will default it to the default max automatically.

Reply:
Gotcha!

Reply:
ttl of `0` looks to be throwing an error:
```
core_portal_django         | tapipy.errors.BadRequestError: message: NET_REQUEST_PAYLOAD_ERROR Error accessing payload for submitJob request: TAPIS_JSON_VALIDATION_ERROR JSON validation error: TAPIS_JSON_VALIDATION_FAILURE JSON validation error: #/subscriptions/0/ttlMinutes: 0 is not greater or equal to 1  #1#/subscriptions/0/ttlMinutes: 0 is not greater or equal to 1
```